src/deskstar-frontend/components/DesksTable.tsx

```diff
@@ -0,0 +1,106 @@
+import { IDesk } from "../types/desk";
+import React from "react";
+
+const DesksTable = ({ desks }: { desks: IDesk[] }) => {
+  return (
+    <div className="overflow-x-auto">
+      <table className="table table-zebra w-full">
+        <thead>
+          <tr>
+            {/* set size of Desk column */}
+            <th className="w-1/4 bg-deskstar-green-light text-left">Desk</th>
+            <th className="bg-deskstar-green-light text-left">Type</th>
+            <th className="bg-deskstar-green-light"></th>
+          </tr>
+        </thead>
+        <tbody>
+          {desks.map((desk: IDesk) => (
+            <DeskTableEntry key={desk.deskId} desk={desk} />
+          ))}
+        </tbody>
+      </table>
+    </div>
+  );
+};
+
+const DeskTableEntry = ({ desk }: { desk: IDesk }) => {
+  return (
+    <tr className="hover">
+      <td className="text-left font-bold">{desk.deskName}</td>
+      <td className="text-left">{desk.deskType}</td>
+      <td className="text-right">
+        <a href="#book-modal" className="btn btn-success">
+          Book
+        </a>
+        <div id="book-modal" className="modal">
```

Comment:
Hätte den Modal sogar in eine eigene Komponente gehauen. Aber sieht gut aus.

Reply:
Jo hab ich dann auch gedacht, aber dachte wir machen das einfach nächste Woche nochmal richtig